client/me/account/main.jsx

```diff
@@ -56,15 +55,14 @@ const user = _user();
 /**
  * Debug instance
  */
-let debug = new Debug( 'calypso:me:account' );
+const debug = new Debug( 'calypso:me:account' );
```

Comment:
this looks very old and I don't usually see `new` with `debug`

you might just consider matching a common pattern from Calypso (but this isn't necessary)

```js
import debugFactory from 'debug';

const debug = debugFactory( 'calypso:me:account' );
```